imcsdk/utils/imcinventory.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2016 Cisco Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains APIs to download inventory to a remote location
+"""
+
+import time
+import logging
+log = logging.getLogger('imc')
+
+
+def _get_parent_dn(handle):
+    from imcsdk.imccoreutils import IMC_PLATFORM
+    if handle.platform == IMC_PLATFORM.TYPE_CLASSIC:
+        return 'sys'
+    elif handle.platform == IMC_PLATFORM.TYPE_MODULAR:
+        return 'sys/chassis-1'
+
+
+def inventory_imc(handle, user=None, pwd=None, hostname=None, remote_file=None,
```

Comment:
ok, we can change this one to `inventory_remote` and the client one to `inventory_local`